src/types.ts

```diff
@@ -66,6 +66,7 @@ export interface Result {
   close: () => void
   setEvaluations: (evaluations: Evaluation[]) => void
   registerAPIRequestMiddleware: (middleware: APIRequestMiddleware) => void
+  refreshCache: () => boolean
```

Comment:
Oh yeah, make sure you update the `README.md` file with the details of the new function.

Reply:
I'm adding a new section for integrating with webviews

Reply:
I'd probably treat it a bit more open and only tangentially mention web views. What we're providing here is a method to manually trigger a refresh of evaluations. One use-case might be for web view coming back into focus, but it's not a 1-1 and I'd imagine that using this for web views is more of the edge-case use rather than the standard use case.

Reply:
I'll move it into its own page so its not front and center